moveit_ros/planning_interface/test/python_move_group_planning.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
```

Comment:
The file tests the pilz_planner, so I would prefer to keep it [with the other pilz tests](https://github.com/ros-planning/moveit/tree/master/moveit_planners/pilz_industrial_motion_planner/test), in contrast to Felix's original post.